src/rulesCheck.ts

```diff
@@ -58,6 +58,7 @@ export const checkRules = (descriptor: SFCDescriptor, filePath: string, apply: s
     functionSize: () => checkFunctionSize(script, filePath, limits.maxFunctionSize),
     htmlImageElements: () => getIsNuxt() && checkHtmlImageElements(descriptor.template, filePath),
     htmlLink: () => isVueFile && checkHtmlLink(descriptor.template, filePath),
+    hugeFiles: () => isVueFile && checkHugeFiles(descriptor, filePath),
```

Comment:
Can we do it to all files, composables, api routes, etc?

Reply:
Yes we can, I'll just have to add a new parameter so the check function knows if it's a Vue or js/ts file to modify the total lines calc